Abstract Syntax and Semantics Document/index.html

```diff
@@ -156,40 +156,57 @@
     </ol>
-    
-    <p>Limitations of the definition:</p>
-    
-    <ul>
-      <li>does <strong>not</strong> allow the default graph of a timestamped graph to have more than triple.</li>
-    </ul>
-    
-    <blockquote>
-      <p><strong>Discussion:</strong> There could be multiple timestamps associated with a graph <code>g</code>, e.g. a start time and an end time, or a generated time and a system processing time. The predicate <code>p</code> should be drawn from a community agreed vocabulary (<a href="https://github.com/streamreasoning/RSP-QL/issues/10">Issue 10</a>).</p>
+      <p class="note">There may be multiple triples in the default graph, including multiple triples using timestamp predicates, 
+        but exactly one triple must be designated as the timestamp triple.</p>
+    
+    <p class="ednote">This definition does not permit the timestamp to be omitted, which is one of the data structures
+      that is considered to be in-scope by the requirements document.
+    </p>
+      <p class="note">
+      This definition allows the timestamp to be implicit.
+      For example, the timestamp predicate could be an IRI or blank node rather than a literal.
+    </p>
       
-      <p><strong>Discussion:</strong> More than one triple may be necessary to represent the time metadata for each graph.</p>
-    </blockquote>
-    
-    <h4>Example:</h4>
-    
-    <p>The following timestamped graph <code>:g1</code> contains 2 triples that state that Darko and Axel are in the Red Room. The <code>p</code> predicate used in this example is the PROV `<code>prov:generatedAtTime</code>. In this example the named graph <code>:g1</code> contains the data contents (triples). The format in the example follows <a href="http://www.w3.org/TR/trig/">TriG</a>, although does not imply any specific serialization or formatting, it simply shows the data structured according to the RDF stream model. Prefixes (e.g. <code>prov:</code>) are used for readability.</p>
-    
-      <pre class="example highlight" title="Timestamped Graph with IRI Graph Name and Quads"><code>:g1 {:axel :isIn :RedRoom. :darko :isIn :RedRoom} {:g1,prov:generatedAtTime,"2001-10-26T21:32:52Z"}
+      <p class="note">An <em>explicitly</em> timestamped graph is a timestamped graph whose timestamp <code>t</code> is a literal.</p>
+      
+      <p class="ednote">Merge and union of equivalence classes of streams with implicitly timestamped graphs may not be defined. See <a href="#merge-union"></a>.</p>
+          
+      <p class="ednote">The timestamp predicate <code>p</code> may be drawn from a community agreed 
+        vocabulary (<a href="https://github.com/streamreasoning/RSP-QL/issues/10">Issue 10</a>).
+        The timestamp predicate may also be user-defined.
```

Comment:
In either case, it would be useful to give a general definition of what is required of a timestamp predicate (https://github.com/streamreasoning/RSP-QL/issues/26).